lib/main.rb

```diff
@@ -31,8 +31,7 @@ def main
       id = gets.chomp.to_i
       app.list_rentals_for_person(id)
     when '7'
-      puts 'Goodbye!'
-      break
+      abort('Goodbye!')
     else
```

Comment:
- [ ] Great work so far but I guess you misunderstood pair-programming here because in pair-programming all the students shoulds should have contributed at least a major requirement instead of just pushing just one line of code and i can see that @Grandi0z has just pushed one line of code which doesn't satisfy the pair-programming requirement 🤔